src/lib/degradeableItems.ts

```diff
@@ -125,7 +141,9 @@ export const degradeableItems: DegradeableItem[] = [
 	{
 		item: getOSItem('Amulet of blood fury'),
 		settingsKey: 'blood_fury_charges',
-		itemsToRefundOnBreak: new Bank().add('Amulet of fury'),
+		run: () => {
+			return new Bank().add('Amulet of fury');
```

Comment:
There is no reason to use anonymous arrays here; these should remain constant Bank() objects, and a union type + type checking should be used to support both types.